nautobot/ipam/factory.py

```diff
@@ -361,9 +346,6 @@ class Meta:
         factory.Maybe("has_location", random_instance(Location, allow_null=False), None),
     )
 
-    has_role = factory.Faker("pybool")
```

Comment:
By removing `has_role` you've changed the behavior here - instead of 50% of VLANs having a `role` and 50% having none, now all of them have a `role`. Reduces the diversity of our test data; I'm against this change. Same comment on the Prefix factory below.